test/rules/Lambda.test.ts

```diff
@@ -117,6 +127,157 @@ describe('AWS Lambda', () => {
     });
   });
 
+  describe('LambdaEventSourceSQSVisibilityTimeout: SQS queue visibility timeout of Lambda Event Source Mapping is at least 6 times timeout of Lambda function', () => {
+    const ruleId = 'LambdaEventSourceSQSVisibilityTimeout';
+    const defaultLambdaFunctionTimeoutSeconds = 3;
+    const defaultSQSVisibilityTimeoutSeconds = 30;
+    const minValidMultiplier = 6;
+    test('Noncompliance 1 - all values defined', () => {
+      const testVisibilityTimeoutSeconds = 20;
+      const lambdaFunction = new Function(stack, 'rFunction', {
+        code: Code.fromInline('hi'),
+        timeout: Duration.seconds(
+          Math.ceil(testVisibilityTimeoutSeconds / minValidMultiplier + 1)
+        ),
+        handler: 'index.handler',
+        runtime: Runtime.NODEJS_20_X,
+      });
+      const sqsQueue = new Queue(stack, 'rSqsQueue', {
```

Comment:
Please remove the `r` prefix from the resource names. That's a legacy naming convention that I'd like to remove from the code base

Reply:
done with b5ebb10